hc/front/tests/test_add_pushbover.py

```diff
@@ -38,3 +39,15 @@ def test_it_validates_nonce(self):
         assert r.status_code == 403
 
     ### Test that pushover validates priority
+    def test_validate_priority(self):
```

Comment:
@collinmutembei pushover is a service that provides a simple way to send notifications to devices from our web app, the notification sent have priority ranging from -2 to 2, this function test that we can only add pushover notification in this valid range so anything else should not be validated ie strings and integers out of range. If the user tries to add such a pushover without the correct priority the response status code should be 400 (bad request). Hope am clear enough we can expand further on our reasoning in our stand up meeting.